packages/nextjs/components/lists/SharedProjects.tsx

```diff
@@ -57,7 +94,7 @@ const SharedProjects: React.FC<Props> = ({ list }) => {
                   alt="project list"
                   height={"80"}
                   width={"80"}
-                  src="/assets/gradient-bg.png"
+                  src={`${project?.profileImageUrl ? project.profileImageUrl : "/assets/gradient-bg.png"}`}
```

Comment:
The images are still not showing up in this component.

Reply:
this is because there is no image for now, once there is image, it will work